simplerunner.lua

```diff
@@ -35,6 +35,21 @@ local function readfile(path)
     return success, text
 end
 
+local function writefile(path, data)
```

Comment:
This function appears to sometimes return `(bool, String)` and sometimes just `bool`. That seems like a recipe for confusion...

Reply:
The non error throwing way of reporting errors in Lua is to return false and the error message or true, and the expected result but theres not always something to return

Reply:
Surely it's better to always return a pair, with null (or similar) in the second position if there's nothing better?